harmony-fe/src/pages/posts/[postId].tsx

```diff
@@ -0,0 +1,142 @@
+import { useRouter } from 'next/router'
+import { Button, Stack, Divider, Paper, Grid } from '@mui/material'
+
+export default function PostDetail() {
+  const router = useRouter()
+  const postID = router.query.postId
+
+  return (
+    <>
+      <div style={{ padding: 14 }}>
+        <Paper style={{ padding: '40px 20px' }}>
+          <h1>Title of the post {postID}</h1>
+          <p>By: Username</p>
```

Comment:
In the future, lets make a `Text` component for the paragraphs and headers

like this:
```
const TextBlock = styled(Typography)`
  ...
`
```